rgf/sklearn.py

```diff
@@ -6,12 +6,18 @@
 from glob import glob
 import numpy as np
 from math import ceil
+import numbers
 
 from sklearn.base import BaseEstimator
 from sklearn.base import ClassifierMixin
 from sklearn.base import RegressorMixin
 from sklearn.utils.extmath import softmax
 from sklearn.utils.validation import NotFittedError
+from sklearn.externals import six
+
+_ALGORITHMS = ["RGF", "RGF_Opt", "RGF_Sib"]
+_LOSSES = ["LS", "Expo", "Log"]
+_FLOATS = (float, np.float, np.float16, np.float32, np.float64, np.double)
 
 sys_name = platform.system()
 WINDOWS = 'Windows'
```

Comment:
Good point!
Don't pay attention to it - next PR will delete these constants with the aim to support MacOS, and another PR will add some encapsulation (like underscores, `__ALL__`, etc.).